### PR TITLE
Obscure location added

### PR DESCRIPTION
**Pull request by mridulnagpal**

### CODE REVIEW

**Comment by mridulnagpal** [comment]
closes #19 Please have a look

File: src/blurredLocation.js [hunk 1]
```diff
@@ -152,6 +153,25 @@ BlurredLocation = function BlurredLocation(options) {
...
+        gridSystem.removeGrid();
+      }
+  }
+
+  function isBlurred() {
+    return blurred;
+  }
+
+  function obscureLocation() {
+    setBlurred(document.getElementById("obscureLocation").checked);
```
Comment (jywarren) [inline src/blurredLocation.js]: OK but this is UI code so lets merge this into the new UI module -- merge that in first.

File: src/blurredLocation.js [hunk 2]
```diff
@@ -152,6 +153,25 @@ BlurredLocation = function BlurredLocation(options) {
...
+  }
+
 
   return {
     getLat: getLat,
```
Comment (jywarren) [inline src/blurredLocation.js]: So, `getLat()` should then return full precision results if `blurred` is false.

File: src/blurredLocation.js (hunk 1 above)
Comment (mridulnagpal) [inline src/blurredLocation.js]: yeah sure

File: src/blurredLocation.js (hunk 2 above)
Comment (mridulnagpal) [inline src/blurredLocation.js]: On it

Comment (mridulnagpal) [inline src/blurredLocation.js]: Done

**Comment by jywarren** [comment]
Super. Done!

**Comment by mridulnagpal** [comment]
Merging cheers!!

**Pull request merged**